### PR TITLE
Lib consolidation

### PR DESCRIPTION
**Pull request by malcolmkiano**

### CODE REVIEW

File: bin/interpreter/lib/index.js [hunk 1]
```diff
@@ -34,20 +36,18 @@ const unga = require('./type/mod/list/unga');
 
 // >>> Strings
 const tenga = require('./type/mod/string/tenga');
+const herufiNdogo = require('./type/mod/string/herufiNdogo');
```
Comment (W3NDO) [inline bin/interpreter/lib/index.js:39]: toLower Function?

File: bin/interpreter/types/boolean.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ class SWBoolean extends SWValue {
   constructor(value) {
     super();
     this.value = value;
+    this.typeName = 'Hali';
```
Comment (W3NDO) [inline bin/interpreter/types/boolean.js:14]: Booleans type is called Hali?

File: bin/interpreter/lib/index.js (hunk 1 above)
Comment (malcolmkiano) [reply bin/interpreter/lib/index.js:39]: Yep

File: bin/interpreter/types/boolean.js (hunk 2 above)
Comment (malcolmkiano) [reply bin/interpreter/types/boolean.js:14]: I guess lol. Couldn't think of anything else

**Review by W3NDO — APPROVED** [review]

File: bin/interpreter/types/boolean.js (hunk 2 above)
Comment (W3NDO) [reply bin/interpreter/types/boolean.js:14]: smart

**Pull request merged**